api/nakadi-event-bus-api.yaml

```diff
@@ -1260,3 +1272,51 @@ definitions:
         description: Documentation for the enrichment.
     required:
       - name
+
+
+  BatchItemResponse:
+    description: |
+      A status corresponding to one individual Event's publishing attempt.
+    properties:
+      eid:
+        type: string
+        format: uuid
+        description: |
+          eid of the corresponding item.
+      publishing_status:
+        type: string
+        enum:
+          - FAILED
+          - SUBMITTED
+          - ABORTED
+        description: |
+          Indicator of the submission of the Event within a Batch.
+          - SUBMITTED indicates successful submission, including commit on he underlying broker.
+          - FAILED indicates the message submission was not possible and can be resubmitted if so desired.
+          - ABORTED indicates that no further attempt at publishing the affected item was attempted because
```

Comment:
I understand the definition, but what do you think of "indicates that this item has not been processed due to a failure on a previous item in the batch".